src/uk/gov/hmcts/contino/GithubAPI.groovy

```diff
@@ -13,22 +13,94 @@ class GithubAPI {
     this.steps = steps
   }
 
+  private static cachedLabelList = [
+    'isValid': false,
+    'cache': []
+  ]
+
+  def currentProject() {
+    return new RepositoryUrl().getShort(this.steps.env.CHANGE_URL)
+  }
+
+  def currentPullRequestNumber() {
+    return this.steps.env.CHANGE_ID
+  }
+
   /**
-   * Add labels to the current pull request.  MUST be run with an onPR() closure.
-   *
-   * @param labels
-   *   A List of labels
+   * Check for Pull Request
    */
-  def addLabelsToCurrentPR(labels) {
+  private isPR(String branch_name) {
+    def isPr = new ProjectBranch(branch_name).isPR()
+    this.steps.echo "Is ${branch_name} a PR?: ${isPr}"
```

Comment:
is this still needed?

Reply:
No - the isPR() method has now been removed.